simulation_parameters/Constants.cs

```diff
@@ -435,6 +435,18 @@ public static class Constants
     /// </summary>
     public const float MUCILAGE_COOLDOWN_TIMER = 1.5f;
 
+    /// <summary>
+    ///   How many time cell gets slowed down with mucocyst (slime jet upgrade) on
+    /// </summary>
+    public const float MUCOCYST_SPEED_DIVISOR = 60;
+
+    public const float MUCOCYST_SPEED_MINIMUM_MUCILAGE = 0.2f;
```

Comment:
```suggestion
    public const float MUCOCYST_MINIMUM_MUCILAGE = 0.2f;
```
Should this variable be named like this (so removing the speed part of the name)?